scripts/pixels.mjs

```diff
@@ -60,11 +101,58 @@ Hooks.on("init", function() {
   module.debounceRoll = foundry.utils.debounce(api.completePendingRoll, 1000);
 });
 
+/* -------------------------------------------- */
+/*  Add Pixels Control Button                   */
+/* -------------------------------------------- */
+
+Hooks.on('getSceneControlButtons', (controls) => {
```

Comment:
Scene controls are not an appropriate place for this sort of button, since it has nothing to do with the scene. If the user has the canvas disabled, for example, all scene controls are also disabled.